modules/button/react/lib/types.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * The different button sizes.
  */
-export enum ButtonSizes {
+export enum ButtonSize {
```

Comment:
Should we convert these to object exports instead of enums and update the the API guidelines to ban the use of enums? (mentioned in https://github.com/Workday/canvas-kit/issues/51)

The reasoning is enums are Typescript-only. The enum is completely compiled away during the Typescript process (requires multi-file compilation) and are not supported in Javascript or the `@babel/plugin-transform-typescript` plugin.

This PR would be a good place to do that.